### PR TITLE
RD-2078 Added system test for Deploy On

### PR DESCRIPTION
**Pull request by kubama**
Additional change: aligned `display_name` property to use recently added templating support.

I verified the system test locally, not planning full system tests execution.

### CODE REVIEW

File: test/cypress/integration/widgets/deployments_view_spec.ts [hunk 1]
```diff
@@ -778,5 +782,58 @@ describe('Deployments View widget', () => {
                 cy.contains('Close').click();
             });
         });
+
+        it('should allow to create child deployments on filtered deployments', () => {
+            cy.interceptSp('POST', '/searches/deployments').as('searchDeployments');
+            cy.interceptSp('PUT', '/deployment-groups/BATCH_ACTION_').as('createDeploymentGroup');
+            cy.interceptSp('POST', '/execution-groups').as('startExecutionGroup');
```
Comment (qooban) [inline test/cypress/integration/widgets/deployments_view_spec.ts:789]: Optional:

I'm wondering - as those two are common for both tests, maybe we can move it one level up to `before` function?

File: test/cypress/integration/widgets/deployments_view_spec.ts [hunk 2]
```diff
@@ -778,5 +782,58 @@ describe('Deployments View widget', () => {
...
+
+            useDeploymentsViewWidget({ configurationOverrides: { filterId: siteFilterName } });
+            widgetHeader.openDeployOnModal();
+
+            const labelKey = 'label_key';
+            const labelValue = 'label_value';
+            cy.get('.modal').within(() => {
+                cy.contains('.field', 'Blueprint').find('input').type(`${blueprintName}{enter}`);
+
+                cy.contains('.field', 'Labels').find('.selection').click();
```
Comment (qooban) [inline test/cypress/integration/widgets/deployments_view_spec.ts:799]: Nice way of selecting dropdown and changing the value 👍🏻 
I see I could have done the same in my test above.

File: test/cypress/integration/widgets/deployments_view_spec.ts [hunk 3]
```diff
@@ -778,5 +782,58 @@ describe('Deployments View widget', () => {
...
+                cy.get('div[name=labelKey] > input').type(labelKey);
+                cy.get('div[name=labelValue] > input').type(labelValue);
+                cy.get('.add').click();
```
Comment (qooban) [inline test/cypress/integration/widgets/deployments_view_spec.ts:802]: Optional:

You can also use `aria-label` in case of Add button in LabelsInput.

**Review by qooban — APPROVED** [review]
Looks good 👍🏻 Added only some optional comments.

File: test/cypress/integration/widgets/deployments_view_spec.ts (hunk 3 above)
Comment (kubama) [reply test/cypress/integration/widgets/deployments_view_spec.ts]: Done

File: test/cypress/integration/widgets/deployments_view_spec.ts (hunk 1 above)
Comment (kubama) [reply test/cypress/integration/widgets/deployments_view_spec.ts]: Done, although it had to be placed in `beforeEach`

**Review by qooban — APPROVED** [review]

File: widgets/common/src/deploymentsView/header/DeployOnModal.tsx [hunk 4]
```diff
@@ -38,7 +38,7 @@ const DeployOnModal: FunctionComponent<DeployOnModalProps> = ({ filterRules, too
                 visibility: deploymentParameters.visibility,
                 new_deployments: environments.map(environmentId => ({
                     id: '{uuid}',
-                    display_name: `${deploymentParameters.blueprintId}-${Stage.Utils.uuid()}`,
+                    display_name: '{blueprint_id}-{uuid}',
```
Comment (tehasdf) [inline widgets/common/src/deploymentsView/header/DeployOnModal.tsx:41]: :+1:

**Pull request merged**